lib/enums/icinga/icinga_objects.py

```diff
@@ -0,0 +1,8 @@
+from enum import auto
+from enums.auto_name import _AutoName
+
+
+# pylint: disable=too-few-public-methods
+class IcingaObject(_AutoName):
```

Comment:
Your using a private implementation detail from Enum incorrectly here unfortunately

It should be like [this example](https://docs.python.org/3/library/enum.html#enum.Enum._generate_next_value_)
```
from enum import auto, Enum

class IcingaObject(Enum):
    HOST = auto()
```